bib/cmd/bootc-image-builder/image.go

```diff
@@ -16,24 +19,28 @@ import (
 	"github.com/osbuild/images/pkg/runner"
 )
 
-func Manifest(imgref string, imgType string, config *BuildConfig, repos []rpmmd.RepoConfig, architecture arch.Arch, seed int64) (*manifest.Manifest, error) {
-
-	source := rand.NewSource(seed)
+type ManifestConfig struct {
+	Imgref       string
+	ImgType      string
+	Config       *BuildConfig
+	Repos        []rpmmd.RepoConfig
+	Architecture arch.Arch
+	TLSVerify    bool
+}
```

Comment:
if these are not meant to be exported (and they are in cmd, so i think not), do they need capitalization? And if so, can you add a comment above each with a brief description of what they are as well as a comment descriptor of ManifestConfig?

Reply:
> if these are not meant to be exported (and they are in cmd, so i think not), do they need capitalization?

I think I would like to move most of the code to a library under `pkg` in a follow-up. Let's figure this out when we have more stable project layout. :)

> And if so, can you add a comment above each with a brief description of what they are as well as a comment descriptor of ManifestConfig?

Done!

Reply:
This can be a follow-up - maybe move files to a pkg/ dir so they can be imported elsewhere?

Reply:
ha! should have refreshed b4 commenting above :)